scripts/can_bundle.py

```diff
@@ -44,9 +44,17 @@ def main(argv):
     default_catalog = config.default_catalog
     plugin_cls = RepositoryPlugin.load(default_catalog)
     plugin = plugin_cls.create(default_catalog)
-    default_source = str(first(plugin.sources))
+    try:
+        default_source = str(one(plugin.sources))
+    except ValueError as e:
+        if 'too many items in iterable (expected 1)' in e.args:
```

Comment:
Generally, we don't want to depend on a human-readable error message to make decisions. It's too brittle. You don't *have* to use `one` and EAFP. Sometimes it's easier to examine the input first.

Reply:
Sorry... checking the error message here was [my idea](https://github.com/DataBiosphere/azul/pull/2901#discussion_r595499253).